versions.gradle

```diff
@@ -56,9 +58,15 @@ ext.versions = versions
 def deps = [:]
 
 def gecko_view = [:]
-gecko_view.nightly_armv7a = "org.mozilla.geckoview:geckoview-nightly-armeabi-v7a:$versions.gecko_view"
-gecko_view.nightly_arm64 = "org.mozilla.geckoview:geckoview-nightly-arm64-v8a:$versions.gecko_view"
-gecko_view.nightly_x86_64 = "org.mozilla.geckoview:geckoview-nightly-x86_64:$versions.gecko_view"
+gecko_view.nightly_armv7a = "org.mozilla.geckoview:geckoview-nightly-armeabi-v7a:$versions.gecko_view_nightly"
+gecko_view.nightly_arm64 = "org.mozilla.geckoview:geckoview-nightly-arm64-v8a:$versions.gecko_view_nightly"
+gecko_view.nightly_x86_64 = "org.mozilla.geckoview:geckoview-nightly-x86_64:$versions.gecko_view_nightly"
+gecko_view.beta_armv7a = "org.mozilla.geckoview:geckoview-beta-armeabi-v7a:$versions.gecko_view_beta"
+gecko_view.beta_arm64 = "org.mozilla.geckoview:geckoview-beta-arm64-v8a:$versions.gecko_view_beta"
+gecko_view.beta_x86_64 = "org.mozilla.geckoview:geckoview-beta-x86_64:$versions.gecko_view_beta"
+gecko_view.release_armv7a = "org.mozilla.geckoview:geckoview-armeabi-v7a:$versions.gecko_view_release"
+gecko_view.release_arm64 = "org.mozilla.geckoview:geckoview-arm64-v8a:$versions.gecko_view_release"
+gecko_view.release_x86_64 = "org.mozilla.geckoview:geckoview-x86_64:$versions.gecko_view_release"
```

Comment:
I don't think we need the armv7a anymore?

Reply:
correct, I have removed the armv7 dependencies